blog/_posts/2016-11-08-RubyConfPt.md

```diff
@@ -5,10 +5,9 @@ created_at: Tues Nov 08 2016
 permalink: blog/2016-11-08-RubyConfPt
 current: blog
 author: Thea Amanda Kupler, Marie-Luise Kochsiek, Mayar Alaa and Nada Ashraf
-twitter: loadtocode
```

Comment:
sorry for nitpicking, but as explained in the [contribution guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#creating-a-new-blog-post) under **IMPORTANT**, the page won't render nicely if the twitter line is completely removed, so please add it back with only one of the two handles. Thanks!